src/ucs/debug/log.c

```diff
@@ -478,6 +478,31 @@ const char *ucs_log_bitmap_to_str(unsigned n, uint8_t *bitmap, size_t length)
     return buf;
 }
 
+void __ucs_log_wqe(const char *file, unsigned line,
```

Comment:
1. can we unite code with ucs_str_dump_hex ?
2. this function should have more generic name such as __ucs_log_hex_word_dump

Reply:
the function expects a buffer in format used in PRM wqe/cqe/cmdif and output it the same way as wqdump does, str_dump_hex has a different format.

Reply:
what if we add "int show_offset" and "int word_size" parameters?